### DISCUSSION TITLE
Sayı dizisindeki soru işaretli yere

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#305] [Zeka Sorusu] [answered]
13 , 19 , 52 , 99 , 117 , 184 , 208 , 295 , 396 , 432 , (?) , 595 , 736 , 891 , 945

Sayı dizisindeki soru işaretli yere hangi sayı gelmelidir

**Comment by AcelyaP** [comment, marked as answer]
Bir tane daha sayı dizisi var buda 6'nın katı olan iki basamaklı sayılardan oluşuyor.

12, 18, 24, 30, 36, 42, 48, 54, 60, 66, 72, 78, 84, 90 ,96

Bu dizideki her sayıyı kendi onlar basamağı çarpanı ile toplayıp, daha sonra bulduğumuz sayıyı da yine aynı onlar basamağı çarpanıyla çarparak ikinci diziyi oluşturuyoruz. Bize 11. yi sorduğu için işlemimizi 72 ile yapıyoruz.

**Cevap:** (72+7)*7=553
